packages/core/utils/src/accountParser.js

```diff
@@ -31,7 +31,20 @@ export default class AccountParser {
         accounts.push(account);
       });
     }
-    return accounts;
+    // Clean up accounts duplicated
+    return accounts.filter((acct, index) => {
+      const sameAccountIndex = accounts.findIndex((acct2, index2) => {
+        if (index === index2) {
+          return false;
+        }
+        const addr1 = acct.address || acct;
```

Comment:
Should the addresses be normalized with `web3.utils.toChecksumAddress` before being compared?

Reply:
Good catch, yes you're right